dags/load_dag_redshift.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import print_function
+
+import logging
+import os
+from datetime import datetime, timedelta
+
+from airflow import models
+from airflow.hooks.postgres_hook import PostgresHook
+from airflow.operators.python_operator import PythonOperator
+
+logging.basicConfig()
+logging.getLogger().setLevel(logging.DEBUG)
+
+
+default_dag_args = {
+    'depends_on_past': False,
+    'start_date': datetime(2015, 7, 30),
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 5,
+    'retry_delay': timedelta(minutes=5)
+}
+
+notification_emails = os.environ.get('NOTIFICATION_EMAILS')
+if notification_emails and len(notification_emails) > 0:
+    default_dag_args['email'] = [email.strip() for email in notification_emails.split(',')]
+
+# Define a DAG (directed acyclic graph) of tasks.
+dag = models.DAG(
+    dag_id='ethereumetl_load_dag_redshift',
+    # Daily at 1:30am
+    schedule_interval='30 1 * * *',
+    default_args=default_dag_args)
+
+dags_folder = os.environ.get('DAGS_FOLDER', '/usr/local/airflow/dags/ethereum-etl-airflow/dags')
+
+
+# Check for required env vars
+output_bucket = os.environ.get('OUTPUT_BUCKET')
+if output_bucket is None:
+    raise ValueError('You must set OUTPUT_BUCKET environment variable')
+
+aws_access_key_id = os.environ.get('AWS_ACCESS_KEY_ID')
+if aws_access_key_id is None:
+    raise ValueError('You must set AWS_ACCESS_KEY_ID environment variable')
+
+aws_secret_access_key = os.environ.get('AWS_SECRET_ACCESS_KEY')
+if aws_secret_access_key is None:
+    raise ValueError('You must set AWS_SECRET_ACCESS_KEY environment variable')
+
+
+def load_task(ds, **kwargs):
+    conn_id = kwargs.get('conn_id')
+    file_format = kwargs.get('file_format')
+    task = kwargs.get('task')
+    pg_hook = PostgresHook(conn_id)
+
+    if file_format == 'csv':
+        sql = """
+            COPY {schema}.{table}
```

Comment:
To make this operation idempotent you may need to delete the records for the current `ds` before loading the file.

Reply:
@medvedev1088 - Addressed by [bbfb7684f37337363db0894a7b8e65c6488f4a22](https://github.com/iter-io/ethereum-etl-airflow/commit/bbfb7684f37337363db0894a7b8e65c6488f4a22)